src/elements/Image/Image.tsx

```diff
@@ -24,10 +26,17 @@ const getContainerStyle = (node: ImageNode): CSSProperties => {
     return { width };
 };
 
-export const Image: FunctionComponent<Props> = ({ children, className, node, ...props }) => {
+export const Image: FunctionComponent<Props> = ({
+    children,
+    className,
+    node,
+    onDownload,
+    onPreviewOpen,
+    ...props
+}) => {
     const { file, href, layout } = node;
     const [isPreviewOpen, setIsPreviewOpen] = useState<boolean>(false);
-    const image = UploadcareImage.createFromPrezlyStoragePayload(file);
+    const image = useMemo(() => UploadcareImage.createFromPrezlyStoragePayload(file), [file.uuid]);
```

Comment:
Hm... Do you think there can be a problem when there are multiple images with the same `uuid`, but different effects/cropping applied?